test/parallel/test-event-emitter-symbols.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+require('common');
+const EventEmitter = require('events');
+const assert = require('assert');
+
+const ee = new EventEmitter();
+const foo = Symbol('foo');
+let didRun = false;
+
+function listener() {
+  didRun = true;
+}
+
+ee.on(foo, listener);
+assert.deepEqual(ee.listeners(foo), [listener]);
+
+ee.emit(foo);
+assert(didRun);
+
+ee.removeAllListeners();
+assert.deepEqual(ee.listeners(foo), []);
+
+ee.on(foo, listener);
+ee.removeListener(foo, listener);
```

Comment:
Not strictly necessary since you tested it above, but maybe add an assertion here that the listener was added.